Cargo.toml

```diff
@@ -12,7 +12,4 @@ proc-macro = true
 [dependencies]
 proc-macro2 = "0.3"
 quote = "0.5"
-syn = "0.13"
-
-[workspace]
-members = ["testcrate"]
+syn = "0.13"
```

Comment:
Why is this change necessary?

Reply:
When the workspace is set up I see the following:

```
 cargo test
   Compiling libc v0.2.39
   Compiling kernel32-sys v0.2.2
   Compiling derive-new v0.5.3
   Compiling serde_derive v1.0.37
   Compiling term v0.4.6
error[E0599]: no method named `make_where_clause` found for type `syn::Generics` in the current scope
  --> github.com-1ecc6299db9ec823\serde_derive-1.0.37\src\bound.rs:47:14
   |
47 |     generics.make_where_clause()
   |              ^^^^^^^^^^^^^^^^^
```

I looked into it a bit but didn't root cause it. Might not occur on a *nix box if the build failed in `term`.

Reply:
This is an error in serde_derive, and it would be weird to see that only on some platforms. I'm on Mac OS and build without error. Maybe it is due to how you are running the tests? I run `cargo test --all` from the root directory, rather than anything in testcrate

Reply:
Yeah, that seems unlikely. I see the same errors when I execute tests that way.

Reply:
ping @dtolnay any idea what might be going on with this Serde build error?

Reply:
No idea but it seems like the sort of thing that `cargo update` (or not having a lockfile checked in) would fix.

Reply:
Looks like you're right, added it back.